swagger/v1/template.yml

```diff
@@ -0,0 +1,24 @@
+---
+openapi: 3.0.1
+info:
+  version: 0.0.1
+  title: "Manage teacher CPD - lead provider API"
+  contact:
+    name: DfE
+    email: continuing-professional-development@digital.education.gov.uk
+  description: "The lead provider API for DfE's manage teacher CPD service"
+servers:
```

Comment:
Like to have a localhost:3000 server for local testing with different scenarios, but easy enough to add and regenerate (just have to remember not to leave it in here).

Reply:
Add "/" as current server